### PR TITLE
:lipstick: feat: change about me font family and style

### LINKED ISSUE #107: Remove the italic style from the about me text and put a beautiful font type
**Describe the bug**
Now there is serif font style in the about me text

**To Reproduce**
1. Go to 'Yoshitha Site'
3. Scroll down to 'About Section'

**Expected behavior**
Remove the italic style from the about me text and put a beautiful font type

**Screenshots**

image

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by AlgyJr**
## Purpose
The purpose of this Pull Request is to fix #107

## Goals
The about me text can be more attractive

## Approach
Just remove the italic font style and change the font family using css import to import a Google Font, Kanit to make the text beautiful

### Screenshots
image

  
### Preview Link
https://pr-108-NEW-YOSHITHA-WEBSITE.surge.sh

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: dark/assets/css/styles.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* Edo Font */
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1]: Some early changes have been commited. Remove them @AlgyJr . Otherwise its perfect.

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css (hunk 1 above)
Comment (AlgyJr) [reply dark/assets/css/styles.css]: Done

**Comment by YoshithaRathnayake** [comment]
No no undo that change @AlgyJr

**Comment by YoshithaRathnayake** [comment]
I told that there are early commits delete them

File: dark/assets/css/styles.css [hunk 2]
```diff
@@ -1,6 +1,5 @@
-/* Edo Font */
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1]: Undo these deleted changes.

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Delete the early commits by creating a new branch, reset the branch, get a pull from the upstream, do, add, commit and push the changes.

File: dark/assets/css/styles.css (hunk 1 above)
Comment (AlgyJr) [reply dark/assets/css/styles.css]: Sorry, I can't understand what you want. Can you detail more?

File: dark/assets/css/styles.css [hunk 3]
```diff
@@ -1,6 +1,7 @@
 /* Edo Font */
 
 @import url('http://fonts.cdnfonts.com/css/edo');
+@import url('https://fonts.googleapis.com/css2?family=Kanit&display=swap');
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:4]: Create another comment called Kanit Font and put the font tag inside the comment

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css (hunk 3 above)
Comment (AlgyJr) [reply dark/assets/css/styles.css:4]: Ok

Comment (AlgyJr) [reply dark/assets/css/styles.css:7]: > Delete the early commits by creating a new branch, reset the branch, get a pull from the upstream, do, add, commit and push the changes.

So I would need to open a new pull request?

Comment (AlgyJr) [reply dark/assets/css/styles.css:7]: @YoshithaRathnayake ?

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:7]: > > Delete the early commits by creating a new branch, reset the branch, get a pull from the upstream, do, add, commit and push the changes.
> 
> So I would need to open a new pull request?

No no now you have already created a branch called dev no so you have to reset it by `git reset --hard upstream/master` and get a pull from the upstream as `git pull upstream master` and do the changes and commit and push

Comment (AlgyJr) [reply dark/assets/css/styles.css:7]: image

@YoshithaRathnayake

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:7]: Ah sorry I couldn't remember one thing wait I will find that and send it to you @AlgyJr

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:7]: `git remote add upstream https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE` and `git fetch upstream` after that the changes in the above comment @AlgyJr

Comment (AlgyJr) [reply dark/assets/css/styles.css:7]: image

This is the project now, how is suppost to do the changes missing files, I can't find the page file you request to change.

Comment (AlgyJr) [reply dark/assets/css/styles.css:7]: @YoshithaRathnayake

Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:7]: can you delete the forked folder and do this process again if you can bcs that's the only way @AlgyJr

Comment (AlgyJr) [reply dark/assets/css/styles.css:7]: image

Comment (AlgyJr) [reply dark/assets/css/styles.css:7]: > can you delete the forked folder and do this process again if you can bcs that's the only way @AlgyJr

Ok

**Pull request closed without merging**